tests/scripts/task_rust.sh

```diff
@@ -13,11 +13,11 @@ cargo fmt -- --check
 
```

Comment:
Please add the frontend resnet as an integration test.

Reply:
it's way too heavy for an integration test since it also involves downloading images and synsets. It doesn't even validate that the resnet model even works using an assert. I think that there's a happy medium somewhere, but as it stands, `resnet` is not a test.

Reply:
Of course resnet works! you're asking if mxnet resnet is valid and it is. Potentially we could cache the cat image, synset and mxnet download.

Reply:
oh yeah, thanks for reminding me why it's disabled. The problem is that `mxnet` isn't installed in the CI docker image. You're welcome to address that as another PR since it wasn't updated in this PR.